docs/quick-start.md

```diff
@@ -2,10 +2,10 @@
 
 ## Requirements
 
-- Install [ckb](https://github.com/nervosnetwork/ckb)(v0.40.0 or above), run ckb and ckb-miner
-- Install [ckb indexer](https://github.com/nervosnetwork/ckb-indexer)(v0.2.1 or above) and run
-- Install [ckb_cli](https://github.com/nervosnetwork/ckb-cli)(v0.40.0 or above)
-- Install [moleculec](https://github.com/nervosnetwork/molecule)(v0.6.1 or above)
+- Install [ckb](https://github.com/nervosnetwork/ckb)(v0.100.0 or above), run ckb and ckb-miner
+- Install [ckb indexer](https://github.com/nervosnetwork/ckb-indexer)(v0.3.0 or above) and run
+- Install [ckb_cli](https://github.com/nervosnetwork/ckb-cli)(v0.100.0 or above)
+- Install [moleculec](https://github.com/nervosnetwork/molecule)(v0.7.2 or above)
```

Comment:
```suggestion
- Install [moleculec](https://github.com/nervosnetwork/molecule)(v0.7.2)
```

https://github.com/nervosnetwork/godwoken/blob/485667bdb666a9ed3ed4b7ec8f6ca9596e0d4cc1/crates/types/build.rs#L21
Seems moleculec should equal to v0.7.2